powershell_shortcut-feedstock/recipe/menu-windows-ps.json

```diff
@@ -5,7 +5,8 @@
         {
             "name": "Anaconda Powershell Prompt",
             "system": "%windir%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe",
-            "scriptarguments": ["-ExecutionPolicy", "ByPass", "-NoExit", "-Command", "& { ${ROOT_PREFIX}\\shell\\condabin\\conda-hook.ps1 ; conda activate ${PREFIX} }"],
+C
```

Comment:
What is this C character here?  Is that going to affect anything?

Reply:
That is a typo. I corrected it already on my branch